new/components/basic/buttons/BaseButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { BaseSyntheticEvent } from "react";
+import { Pressable } from "react-native";
+import { BaseMiddleText } from "../layout/BaseText";
+import { customStyleInput } from "@/lib/customStyleApi";
+import { useStyle } from "@/hooks/styles/useTheme";
+
+interface SimpleButtonProps {
+    title: string,
+    styleClassName?: string,
+    onPress?: ((e: BaseSyntheticEvent) => any) | (() => any),
+    textColor?: string,
+}
+export default function BaseButton(props: SimpleButtonProps) {
+    const { onPress, textColor, title = 'Save', styleClassName = "baseButtonStyle" } = props;
+    const styleWithClass = customStyleInput(useStyle(styleClassName), baseButtonStyle);
+
+    return (
+        <Pressable onPress={onPress} style={styleWithClass}>
+            <BaseMiddleText styleClassName={textColor} >{title}</BaseMiddleText>
```

Comment:
In the application, will you support any other pressable customised component.
e.g. a pressable item with paragraph and header, a pressable image, etc..

Reply:
It will be another types of button (ie. another component for 'BaseLargeText' button) but so far i cannot think of any scenarios that requires a large text button so that's why I only created one. If one day we need a large Text one we can simply create another the large Text button on this page i assume, just like how i did on the baseText ones.